README.md

```
@@ -1,166 +1,324 @@
+  [8] ⌚ Device & Technical
+  [9] 🎽 Gear & Equipment
+  [0] 💧 Hydration & Wellness
+  [a] 🔧 System & Export
+
+  [q] Exit program
+
 Make your selection: 
 ```
 
-[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.me/cyberjunkynl/)
+### API Coverage Statistics
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)